packages/cli/src/commands/repl.ts

```diff
@@ -0,0 +1,13 @@
+import {Command, ux} from '@oclif/core'
+
+export default class Repl extends Command {
+  static description = 'enter an interactive REPL session to run Heroku CLI commands'
+
+  static examples = [
+    '$ heroku --repl',
```

Comment:
@justinwilaby Why is this `heroku --repl` instead of just `heroku repl` command?

Reply:
@SandyPantsLai - This was decided on as a temporary flag until a major release is made where repl mode will be just `heroku`. It follows the pattern of `heroku --help`.